src/custom/abis/types/factories/GPv2Settlement__factory.ts

```diff
@@ -2,86 +2,92 @@
 /* tslint:disable */
 /* eslint-disable */
 
-import { Contract, Signer, utils } from 'ethers'
-import { Provider } from '@ethersproject/providers'
-import type { GPv2Settlement, GPv2SettlementInterface } from '../GPv2Settlement'
+import { Contract, Signer, utils } from "ethers";
+import { Provider } from "@ethersproject/providers";
+import type {
+  GPv2Settlement,
+  GPv2SettlementInterface,
+} from "../GPv2Settlement";
 
 const _abi = [
   {
     anonymous: false,
     inputs: [
       {
         indexed: true,
-        internalType: 'address',
-        name: 'owner',
-        type: 'address',
+        internalType: "address",
+        name: "owner",
+        type: "address",
       },
       {
         indexed: false,
-        internalType: 'contract IERC20',
-        name: 'sellToken',
-        type: 'address',
+        internalType: "contract IERC20",
+        name: "sellToken",
+        type: "address",
       },
       {
         indexed: false,
-        internalType: 'contract IERC20',
-        name: 'buyToken',
-        type: 'address',
+        internalType: "contract IERC20",
+        name: "buyToken",
+        type: "address",
       },
       {
         indexed: false,
-        internalType: 'uint256',
-        name: 'sellAmount',
-        type: 'uint256',
+        internalType: "uint256",
+        name: "sellAmount",
+        type: "uint256",
       },
       {
         indexed: false,
-        internalType: 'uint256',
-        name: 'buyAmount',
-        type: 'uint256',
+        internalType: "uint256",
+        name: "buyAmount",
+        type: "uint256",
       },
       {
         indexed: false,
-        internalType: 'uint256',
-        name: 'feeAmount',
-        type: 'uint256',
+        internalType: "uint256",
+        name: "feeAmount",
+        type: "uint256",
       },
       {
         indexed: false,
-        internalType: 'bytes',
-        name: 'orderUid',
-        type: 'bytes',
+        internalType: "bytes",
+        name: "orderUid",
+        type: "bytes",
       },
     ],
-    name: 'Trade',
-    type: 'event',
+    name: "Trade",
+    type: "event",
   },
   {
     inputs: [
       {
-        internalType: 'bytes',
-        name: 'orderUid',
-        type: 'bytes',
+        internalType: "bytes",
+        name: "orderUid",
+        type: "bytes",
       },
       {
-        internalType: 'bool',
-        name: 'signed',
-        type: 'bool',
+        internalType: "bool",
+        name: "signed",
+        type: "bool",
       },
     ],
-    name: 'setPreSignature',
+    name: "setPreSignature",
     outputs: [],
-    stateMutability: 'nonpayable',
-    type: 'function',
+    stateMutability: "nonpayable",
+    type: "function",
   },
-]
+];
 
 export class GPv2Settlement__factory {
-  static readonly abi = _abi
+  static readonly abi = _abi;
```

Comment:
My guess is the linter is told to ignore these files now, but maybe it wasn't the case before.
And since it auto-generates with semi-columns, here they are.

Not a big deal IMO.

Reply:
They say this https://github.com/dethcrypto/TypeChain#q-generated-files-wont-match-current-codestyle-of-my-project- but maybe its not working. As Leandro mentioned this file is generated using typechain.